packages/fxa-payments-server/src/routes/Subscriptions/Reactivate/ManagementPanel.tsx

```diff
@@ -45,10 +38,6 @@ export default ({
     hideReactivateConfirmation,
   ]);
 
-  const cancelledAt = subscription.cancelledAt
```

Comment:
This was one of the last vestiges of data used from the old subscription data, back when we pulled this from an FxA database.

We stopped doing that for issue #3808 and Stripe data doesn't offer this info. 

We permanently hid the message as part of issue #3400. This just goes ahead and removes the message altogether.